tensorflow_serving/model_servers/main.cc

```diff
@@ -352,6 +354,22 @@ tensorflow::serving::PlatformConfigMap ParsePlatformConfigMap(
   return platform_config_map;
 }
 
+tensorflow::Status readkey( const tensorflow::string& filename, tensorflow::string& data )
+{
+    std::ifstream file(filename.c_str (), std::ios::in);
```

Comment:
For consistency, please use the tensorflow file read libraries as we already do elsewhere in this file, e.g. in ParseProtoTextFile().